test/functional/cmdLineTests/jitserver/jitserverArgumentTesting.xml

```diff
@@ -89,4 +92,46 @@
 		<output type="failure" caseSensitive="yes" regex="no">JITSERVER NO LONGER EXISTS</output>
 	</test>
 
+	<test id="Test SSL success condition">
+		<command>bash $SCRIPPATH$ $TEST_RESROOT$ $TEST_JDK_BIN$ "$DEFAULT_JITSERVER_OPTIONS$" "$ENABLE_JITSERVER$ $JITSERVER_VERBOSE$ $JITSERVER_SSL1$" false</command>
+		<output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk|semeru) version</output>
+		<output type="required" caseSensitive="no" regex="no">JITServer Client Mode.</output>
+		<output type="success" caseSensitive="no" regex="no">Successfully initialized SSL context</output>
```

Comment:
I remember discussing that the test succeeds when any of the `type="success"` criteria is met. 
If that is the case, then "Successfully initialized SSL context" is sufficient to make the test pass, but in reality that is not enough.

Reply:
In this test we can make that condition `type="required"` for "Successfully initialized SSL context" which will enforce the test to look for this condition and make the test pass only if the condition is met. 
However the test will only search for the condition "Successfully initialized SSL context" in the output of the test which includes logs from the server and the client and the test will pass if either client or server logs contain the text.

I can remove the `-Xjit:verbose={JITServer}` from the server side to avoid checking the logs from the server and change few conditions in the test which currently expects server side logs.

Or a better way might be to make the condition `JITServer::StreamFailure: Failed to SSL_connect` as `required `condition in the failure scenario and for success scenario make the `Connected to a server` as `required` condition.

Reply:
I have removed the server side verbose logging and modified the conditions to check if the connection was success/failure (based on the test criteria) as a `required` condition.